### ISSUE TITLE
install fails with "/usr/local/src//: Is a directory"

### ISSUE THREAD

**Issue by mustmodify** [postmodern/ruby-install#229]
First off, thanks for ruby-install. I wouldn't have thought it would make my life dramatically easier but it seems to help. :)

I just tried this on a different machine and it worked fine. Should be the same setup as I used the same script. :)

```
jw@xyst:~$ ls /opt/rubies/
rbx-2.5.7  ruby-2.2.2

jw@xyst:~$ CC=clang sudo ruby-install rbx 2.5.8
grep: /usr/local/bin/../share/ruby-install/rbx/md5.txt: No such file or directory
>>> Installing rbx 2.5.8 into /opt/rubies/rbx-2.5.8 ...
>>> Installing dependencies for rbx 2.5.8 ...
Reading package lists... Done
Building dependency tree
Reading state information... Done
automake is already the newest version.
bison is already the newest version.
flex is already the newest version.
g++ is already the newest version.
gcc is already the newest version.
libgdbm-dev is already the newest version.
libncurses5-dev is already the newest version.
libreadline-dev is already the newest version.
zlib1g-dev is already the newest version.
llvm-dev is already the newest version.
libssl-dev is already the newest version.
libyaml-dev is already the newest version.
ruby1.9.1-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 14 not upgraded.
>>> Downloading  into /usr/local/src ...
/usr/local/src//: Is a directory
!!! Download of  failed!
```

**Comment by mustmodify** [comment]
this worked:

    root@xyst:~$ ruby-install rubinius 2.5.8

**Comment by mustmodify** [comment]
also unfortunate:

    jw@xyst:~$ chruby rubinius-2.5.8
    Warning:: command not found
    Warning:: command not found
    Warning:: command not found
    Warning:: command not found

**Comment by mustmodify** [comment]
This is just strange.

According to my reading of the code on master, the ruby version comes from [this list](https://github.com/postmodern/ruby-install/blob/e2edd47e7e46b33592a0e990d25fe5046d6cb081/share/ruby-install/ruby-install.sh#L10). That list hasn't been touched since fc290f on 2015-05-06. (and even then, the list didn't change.)

I checked /usr/local/share/ruby-install/ruby-install.sh and it has a different list of options:

    #!/usr/bin/env bash
    
    shopt -s extglob
    
    RUBY_INSTALL_VERSION="0.3.3"
    RUBY_INSTALL_DIR="${BASH_SOURCE[0]%/*}"
    
    RUBIES=(ruby jruby rubinius maglev)
    PATCHES=()
    CONFIGURE_OPTS=()

and also:

```

jw@xyst:/usr/local/share/ruby-install$ find . -type d
.
./mruby
./rubinius
./rbx
./maglev
./jruby
./ruby
./ruby/patches
./ruby/patches/1.8
```

which inevitably leads to:

jw@xyst:/usr/local/share$ ruby-install
```
Known ruby versions:
  ruby:
    1.8:    1.8.7-p374
    1.8.7:  1.8.7-p374
    1:      1.9.3-p448
    1.9:    1.9.3-p484
    1.9.3:  1.9.3-p484
    2:      2.0.0-p353
    2.0:    2.0.0-p353
    2.0.0:  2.0.0-p353
    2.1:    2.1.0-preview2
    2.1.0:  2.1.0-preview2
    stable: 2.0.0-p353
  jruby:
    1.7:    1.7.8
    stable: 1.7.8
  rubinius:
    2.1:    2.1.1
    2.2:    2.2.1
    stable: 2.2.1
  maglev:
    1.0:    1.0.0
    1.1:    1.1RC1
    stable: 1.0.0
```

**Comment by mustmodify** [comment]
adding `rbx` to the list of rubies leads to:

```
jw@xyst:/usr/local/share/ruby-install$ ruby-install
Known ruby versions:
  ruby:
    1.8:    1.8.7-p374
    1.8.7:  1.8.7-p374
    1:      1.9.3-p448
    1.9:    1.9.3-p484
    1.9.3:  1.9.3-p484
    2:      2.0.0-p353
    2.0:    2.0.0-p353
    2.0.0:  2.0.0-p353
    2.1:    2.1.0-preview2
    2.1.0:  2.1.0-preview2
    stable: 2.0.0-p353
  jruby:
    1.7:    1.7.8
    stable: 1.7.8
  rbx:
    2.1.0
    2.1.1
    2.2.0
    2.2.1
    2.2.2
    2.2.3
    2.2.4
    2.2.5
    2.2.6
    2.2.7
    2.2.8
    2.2.9
    2.2.10
  rubinius:
    2.1:    2.1.1
    2.2:    2.2.1
    stable: 2.2.1
  maglev:
    1.0:    1.0.0
    1.1:    1.1RC1
    stable: 1.0.0
```

**Comment by mustmodify** [comment]
However, the issue isn't resolved:

```
jw@xyst:~$ CC=clang ruby-install rbx 2.5.8
grep: /usr/local/bin/../share/ruby-install/rbx/md5.txt: No such file or directory
>>> Installing rbx 2.5.8 into /home/jw/.rubies/rbx-2.5.8 ...
>>> Installing dependencies for rbx 2.5.8 ...
Reading package lists... Done
Building dependency tree
Reading state information... Done
automake is already the newest version.
bison is already the newest version.
flex is already the newest version.
g++ is already the newest version.
gcc is already the newest version.
libgdbm-dev is already the newest version.
libncurses5-dev is already the newest version.
libreadline-dev is already the newest version.
zlib1g-dev is already the newest version.
llvm-dev is already the newest version.
libssl-dev is already the newest version.
libyaml-dev is already the newest version.
ruby1.9.1-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 14 not upgraded.
>>> Downloading  into /home/jw/src ...
/home/jw/src//: Is a directory
!!! Download of  failed!
```

**Comment by havenwood** [comment]
It seems there's an old, pre-0.4.0 version of ruby-install (before [this commit](https://github.com/postmodern/ruby-install/commit/962c41abc85027accf8fee5a3e147acfe80c2238) landed) still lingering on your system.

See if a `sudo make uninstall` and fresh install of ruby-install resolves the issue?

**Comment by havenwood** [comment]
Closing this since the issue has been resolved since version 0.4.0. If you run into this try updating to the latest ruby-install.
